datalake/src/main/resources/schema/app/20210920112855_CB-14194_adding_multiaz_field_to_datalake_object_to_request_a_multiaz_aws_datalake.sql

```diff
@@ -0,0 +1,12 @@
+-- // CB-14194 adding multiaz field to datalake object to request a multiaz aws datalake
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE sdxcluster ADD COLUMN IF NOT EXISTS enablemultiaz boolean;
+UPDATE sdxcluster SET enablemultiaz = FALSE WHERE enablemultiaz is NULL;
+ALTER TABLE sdxcluster ALTER enablemultiaz SET DEFAULT false;
```

Comment:
Insert `COLUMN` between `ALTER` and `enablemultiaz`.

Reply:
?

Reply:
Never mind, `COLUMN` after `ALTER` is entirely optional. Fell free to ignore. :smile: